### ISSUE TITLE
[Bug] ScalarBarActor text background is not transparent

### ISSUE THREAD

**Issue by BishopWolf** [Kitware/vtk-js#2840]
### Bug description

The ScalarBarActor renders text that has no transparent background. In the [definition](https://kitware.github.io/vtk-js/api/Rendering_Core_ScalarBarActor.html) for text style, it does not appear anything to control this behaviour.
This affects the Title and the ticks!
```
export interface IStyle {
	fontColor?: string;
	fontStyle?: string;
	fontFamily?: string;
	fontSize?: string;
}
```

### Steps to reproduce

```
function buildScalarBar(lookupTable) {
		globalScene.scalarBar = vtk.Rendering.Core.vtkScalarBarActor.newInstance();
		globalScene.scalarBar.resetGenerateTicksToDefault();
		globalScene.scalarBar.setScalarsToColors(lookupTable);
		globalScene.scalarBar.setAxisLabel("This text with background");
		globalScene.scalarBar.setDrawNanAnnotation(false);
		globalScene.scalarBar.setDrawBelowRangeSwatch(false);
		globalScene.scalarBar.setDrawAboveRangeSwatch(false);
		globalScene.scalarBar.setAxisTextStyle({
			fontStyle: 'normal',
			fontFamily: 'Arial',
			fontColor: 'white',
			fontSize: '20'
		});
		
		globalScene.renderer.addActor2D(globalScene.scalarBar);

		globalScene.renderWindow.render();
	}
```

### Detailed Behavior

![image](https://github.com/Kitware/vtk-js/assets/11816952/4dda58d7-81ab-4b93-99f5-39e38c78f083)

### Expected Behavior

The text is rendered with transparent background, or at least we have a way to trigger this behaviour.

### Environment

- vtk.js version:
- Browsers:
- OS:

**Comment by finetjul** [comment]
Do you have the same problem with [this](https://kitware.github.io/vtk-js/examples/ScalarBarActor.html) example.
I do not.
![image](https://github.com/Kitware/vtk-js/assets/219044/7b2af72d-5728-493b-9a87-25159627b11f)

**Comment by BishopWolf** [comment]
Using the code from the example in a new node application with all requirements automatically loaded it works as intended.

Using a default scalarBarActor in an enterprise application with bundled vtk.js (no internet access) it works as in my example. I have tried several methods but I can't change the text background. I presume I am missing a required library to be loaded.

**Comment by finetjul** [comment]
Are you using the same version of VTK.js in both cases?

**Comment by BishopWolf** [comment]
vtk.js 28.2.0

**Comment by BishopWolf** [comment]
Using bundled vtk.js 28.2.1
Maybe you have changed something, in that case thanks, it is working now.

![image](https://github.com/Kitware/vtk-js/assets/11816952/8c363f76-3a7c-4789-95dd-f4ed97b1aa1d)

**Comment by finetjul** [comment]
No, we have not changed anything related to that. I'm not sure what happened.
